src/apps_plugin.c

```diff
@@ -2482,7 +2532,7 @@ int main(int argc, char **argv)
 		send_collected_data_to_netdata(groups_root_target, "groups", dt);
 
 		if(debug) fprintf(stderr, "apps.plugin: done Loop No %llu\n", counter);
-		fflush(NULL);
+		fflush(NULL);	// FIXME: Is this really necessary?
```

Comment:
It should not. It is probably a left over of a debugging session. Please remove it.